lib/src/source/git.dart

```diff
@@ -883,6 +979,7 @@ class ResolvedGitDescription extends ResolvedDescription {
     return {
       'url': url,
       'ref': description.ref,
+      'tag_pattern': description.tagPattern,
       'resolved-ref': resolvedRef,
```

Comment:
Should we use `tag-pattern` for consistency?

Reply:
Hmm - this is `pubspec.lock`.
In pubspec.yaml we use `_` (at least in `pubspec_overrides` and `dev_dependencies`.

So when we copy the field from the pubspec.yaml to pubspec.lock it is confusing if we change the spelling.

At the same time it is confusing that we have `resolved-ref` in the pubspec.lock.

Can we find a single-word alternative?